articles/libraries/machine_learning/concepts/Qccc.Introduction.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
As the build bot warning noted, you'll need a YAML header here to provide metadata about the page.
```suggestion
---
title: Quantum machine learning library
author: alexeib2
ms.author: alexei.bocharov@microsoft.com
ms.date: 11/22/2019
ms.topic: article
uid: microsoft.quantum.libraries.machine-learning.index
---
```

Reply:
Hey Alex, could you please take a look at Chris' comments? He left suggestions what to change. If you agree with these suggestions, you can just click on the "+" symbol above (or Ctrl+g) and by magic they will be done. @alexeib2